app/lib/hooks/mockedFontScaleProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { FONT_SCALE_LIMIT, ResponsiveLayoutContext } from './useResponsiveLayout';
+
+type Props = {
+	children: React.ReactNode;
+	fontScale?: number;
+};
+
+export const MockResponsiveFontScaleProvider = ({ children, fontScale = 1.4 }: Props) => {
```

Comment:
This file is strange.
It's not clear what's being mocked since there's no FontScaleProvider to match a mockedFontScaleProvider.
I can see it's being used on message stories, so it's probably more useful to use `jest.mock` inside the stories file.
Mocking `useResponsiveLayout` is also another option.

If you ever need to create a whole file to mock something, you should either create it on `__mocks__` folder or use `.mock.ts` prefix.